config/config.exs

```diff
@@ -0,0 +1,12 @@
+import Config
+
+maybe_get_int = fn var ->
+  case System.get_env(var, "") do
+    "" -> nil
+    value -> String.to_integer(value)
+  end
+end
+
+unless System.get_env("BAKEWARE_ARGC") do
+  System.put_env("BAKEWARE_ARGC", "0")
+end
```

Comment:
Can you point me at the documentation for this? Trying to understand how "maybe_get_int" and BAKEWARE_ARGC are used by bakeware

Reply:
This was done so "iex -S mix" works as intended. Another option is to rework the supervision tree so this module isn't the main app when not in a release

Reply:
Explaining in more depth, because Bakeware is kinda new, this variable is still required. If we try to run Fika.Cli without defining it, it crashes on an Integer.parse error

Reply:
https://github.com/spawnfest/bakeware/pull/49